code/pages/Site.php

```diff
@@ -156,6 +156,9 @@ public function onBeforeWrite() {
 			}
 		}
 
+		//Set MenuTitle to NULL so that Title is used
+		$this->MenuTitle = NULL;
```

Comment:
For Site it isn't exposed in the CMS, in normal use it should remain as NULL but if another module modifies it or if the Site is a conversion from a regular Page, the value will be returned oddly.

The other solution is adding:
```
public function getMenuTitle(){
	return $this->Title;
}
```

Which also the value to remain in the DB.